src/Table/Cell.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {PropTypes} from "react";
+
+import {joinClasses as cls} from "../utils";
```

Comment:
I don't see much of a benefit from aliasing, i'd just use it as `joinClasses` to make it more homogeneous